common/views/components/BannerCard/BannerCard.tsx

```diff
@@ -1,47 +1,57 @@
 import { classNames, font } from '../../../utils/classnames';
 import LabelsList from '../LabelsList/LabelsList';
-// import { formatDate } from '../../../../common/utils/format-date';
 import { trackEvent } from '../../../utils/ga';
 import { FunctionComponent } from 'react';
 import { Season } from '../../../model/seasons';
 import linkResolver from '../../../../common/services/prismic/link-resolver';
 import styled from 'styled-components';
 import Space from '../styled/Space';
-import { UiImage } from '../Images/Images';
+import { convertImageUri } from '../../../utils/convert-image-uri';
 import ButtonOutlined from '../ButtonOutlined/ButtonOutlined';
 
 type CardOuterProps = {
-  highlightColor: 'yellow' | 'orange';
   background: 'charcoal' | 'cream';
 };
 
 const CardOuter = styled.a<CardOuterProps>`
   display: flex;
-  flex-direction: column;
+  flex-direction: column-reverse;
   overflow: hidden;
   text-decoration: none;
-  border-left: 4px solid ${props => props.theme.color(props.highlightColor)};
   background: ${props => props.theme.color(props.background)};
   color: ${props =>
     props.theme.color(props.background === 'charcoal' ? 'cream' : 'black')};
 
   ${props => props.theme.media.large`
     flex-direction: row;
-  `}
+    `}
 `;
 
-const TextWrapper = styled.div`
+type TextWrapperProps = {
+  highlightColor: 'yellow' | 'orange';
+};
+
+const TextWrapper = styled.div<TextWrapperProps>`
   ${props => props.theme.media.large`
     flex-grow: 2;
-  `};
+    `};
+  border-left: 4px solid ${props => props.theme.color(props.highlightColor)};
 `;
-const ImageWrapper = styled.div`
+
+type ImageWrapperProps = {
+  imageUrl: string;
+};
+
+const ImageWrapper = styled.div<ImageWrapperProps>`
+  background-image: url(${props => props.imageUrl});
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center top;
+  height: 300px;
   ${props => props.theme.media.large`
-    flex-grow: 1;
-    flex-basis: 106%;
-  `};
-  ${props => props.theme.media.xlarge`
-    flex-basis: 42%;
+    background-position: center center;
+    height: auto;
+    min-width: 38%;
```

Comment:
Out of interest - why was this chosen? Just by eye?

Reply:
yep